src/index.html

```diff
@@ -3,6 +3,138 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <main>
+        <div id="home-page">
+            <!-- Home page content -->
+        </div>
+        
+        <div>
+            <p>pop1</p>
+            <button class="btn1">POPUP1</button>
+        </div>
+        <div>
+            <p>pop2</p>
+            <button class="btn2">POPUP2</button>
+        </div>
+    </main>
+
+    <footer>
+        <!-- Footer content -->
+        <p>&copy; 2023 Food Restaurant</p>
+    </footer>
+    <div id="comments-popup" class="popup">
+        <!-- Comments popup content -->
+    </div>
+    <div id="reservations-popup" class="popup">
+        <!-- Reservations popup content -->
+    </div>
+
+    <!-- ===============================================
+                    Comments Popup Section
+                ==================================================== -->   
+                <section class="popup-section">
+                    <div class="popup-container1 d-none">
+                        <div class="popup-contents">
+            
+                            <span class="close-popupA">X<i class="fa fa-times fa-1.5x"></i></span>
+                            
+                            <div class="popup-imgs-container">
+                                <img src="books-bg.jpg" class="popup-img" alt="img">
+                                <p class="popup-img-description">Space 3</p>
+                            </div>
+            
+                            <section class="popup-descriptive-info">
+                                <div class="desc1">
```

Comment:
These was not supposed to be there at first place its not supposed to be there these all things need to be removed the all line and make it dynamically